plugins/out_stackdriver/stackdriver.c

```diff
@@ -2410,8 +2415,19 @@ static flb_sds_t stackdriver_format(struct flb_stackdriver *ctx,
         }
 
         /* logName */
-        len = snprintf(path, sizeof(path) - 1,
+        project_key_extracted = FLB_FALSE;
```

Comment:
I think the extraction here should be moved up with the rest of the extractions, like line 2212ish.

The `/* logName */` comment should also be moved down to where the log name check is now.